### PR TITLE
[#38] Update readme

### PR DESCRIPTION
**Pull request by karntrehan**
Updating readme to include module integration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,73 +41,124 @@ We have also built a bunch of additional features on top of ODK collect:
...
 
 ## Versions
 
-* Current version: TBA
+* Current version: 0.0.1
     * Based on ODK Release: [v2022.4.4](https://github.com/getodk/collect/releases/tag/v2022.4.4)
 
+## Integration
+
+### via Module
```
Comment (chinmoy12c) [inline README.md:55]: We also need to include that we need to set up google-services.json, ODK Server, and change `APP_PROVIDER` in `ProjectKeys.kt`.

File: README.md [hunk 2]
```diff
@@ -2,12 +2,16 @@
...
+### ODK Collect
+
 ODK Collect is an open source Android app that replaces paper forms used in survey-based data
 gathering. It supports a wide range of question and answer types, and is designed to work well
 without network connectivity.
 
 You can read more about ODK Collect [here](https://docs.getodk.org/collect-intro/).
 
+### ODK Collect Extension
+
```
Comment (chinmoy12c) [inline README.md:14]: We need to list and document the additional settings we have created apart from those provided by the collect app, for example, theme, form flow modifications, etc.

File: README.md [hunk 3]
```diff
@@ -37,73 +41,128 @@ We have also built a bunch of additional features on top of ODK collect:
...
+* **develop** - This branch is used for active development. All feature branches are merged into
+  this branch, and it is updated regularly. This branch is intended to be used for experimental and
+  ongoing development work.
 
-***feature/$FEATURE_NAME:*** These branches are used for active development work, with each branch
-focusing on a specific feature or functionality. They are created from the develop branch, and when
-work is completed, they are merged back into develop.
+* **feature/$FEATURE_NAME** - These branches are used for active development work, with each branch
+  focusing on a specific feature or functionality. They are created from the develop branch, and
+  when work is completed, they are merged back into develop.
```
Comment (chinmoy12c) [inline README.md:161]: We need to document a way to enable contributors to set up a test ODK server and Firebase instance.

File: README.md (hunk 2 above)
Comment (karntrehan) [reply README.md:14]: Custom theming on form screens, Callbacks on form submission etc have been mentioned. Detailed documentation would be in the wiki

File: README.md (hunk 3 above)
Comment (karntrehan) [reply README.md:161]: Good idea. Let me check if ODK has a documented way to do this already.

**Pull request merged**